requirements/lite/base.txt

```diff
@@ -0,0 +1,8 @@
+# NOTE: the upper bound for the package version is only set for CI stability, and it is dropped while installing this package
+#  in case you want to preserve/enforce restrictions on the latest compatible version, add "strict" as an in-line comment
+
+torch>=1.9.*, <1.13.0
+fsspec[http]>=2021.05.0, !=2021.06.0, <2022.6.0
+pyDeprecate>=0.3.1, <=0.3.2
```

Comment:
Why does Lite need pyDeprecate?

In fact, I think it can be removed from PL too

```python
$ grep -iIrn "pyDeprecate" *
docs/source-pytorch/conf.py:339:    "pyDeprecate": "deprecate",
requirements/pytorch/base.txt:11:pyDeprecate>=0.3.1, <=0.3.2
```

Reply:
sure if we do not use anymore, let's drop it :)

Reply:
Done in https://github.com/Lightning-AI/lightning/pull/14472

Reply:
Removed here too